pkg/rtc/transport.go

```diff
@@ -1446,9 +1441,11 @@ func (t *PCTransport) processEvents() {
 
 		err := t.handleEvent(&event)
 		if err != nil {
-			t.params.Logger.Errorw("error handling event", err, "event", event.String())
-			if onNegotiationFailed := t.getOnNegotiationFailed(); onNegotiationFailed != nil {
-				onNegotiationFailed()
+			if !t.isClosed.Load() {
```

Comment:
we are getting errors that operations couldn't be completed due to PeerConnection has closed. so handling that race better